derpycode/cgenie.muffin
# _DEV --> master

Now here is an admission ...

In commit 8e340e5b171b7d86d3e47b0c163200fab623aa67
'revised color and preformed tracer behavior' on 13th May:

In addition to what I was meant to be doing (fixed up the preformed tracer code), I changed the averaging of surface time-series variables, from an ice-free basis, to an entire ocean surface averaging basis.
This changed the values (for climates with sea-ice) and confused the hell out of me as I thought something had gone 'wrong' with experiments post this commit.

I have hence partially un-done this, so that in the same column of sur time-series output as before, the same ice-free ocean surface average appears.
An additional new last row now appears, which is the whole ocean surface average (which should be the same as the ocean-free number only when there is no sea-ice present).

To Test:
(1) checkout
(2) make testbiogem
(3) re-run the first ca. 10 years of any experiment(s) pre-dating 13th May 2019, and confirm that the surface ocean ice-free time-series values are the same.

NOTE: Surface (and benthic) means are only reported for ocn tracers. There is also a specific set of sur carbonate chemistry time-series output. (All depending on what output options are selected.)
NOTE: The 2 different surface averaged columns are distinguished:
'surface (ice-free)' vs. just 'surface'
